app/src/main/java/com/imashnake/animite/features/searchbar/SearchViewModel.kt

```diff
@@ -41,4 +41,8 @@ class SearchViewModel @Inject constructor(
             }
         }
     }
+
+    fun clearList() {
+        uiState = uiState.copy(searchList = null)
```

Comment:
aight I'll be fixing this as well then lmao

Reply:
:kappa: